index.html

```diff
@@ -1167,18 +1174,19 @@ <h3>Algorithm</h3>
                   an <a data-link-for="JsonLdErrorCode">invalid @version value</a>
                   has been detected, and processing is aborted.</li>
                 <li>If <a>processing mode</a>
-                  is set to <code>json-ld-1.0</code>,
+                  is set to `json-ld-1.0`,
                   a <a data-link-for="JsonLdErrorCode">processing mode conflict</a>
                   error has been detected and processing is aborted.</li>
                 <li>Set <a>processing mode</a>,
-                  to <code>json-ld-1.1</code>, if not already set.</li>
+                  to `json-ld-1.1`, if not already set.</li>
               </ol>
             </li>
             <li class="changed">If <var>context</var> has an <code>@import</code> <a>entry</a>:
               <ol>
-                <li>If <a>processing mode</a> is <code>json-ld-1.0</code>,
+                <li>If <a>processing mode</a> is `json-ld-1.0`,
                   an <a data-link-for="JsonLdErrorCode">invalid context entry</a>
-                  error has been detected and processing is aborted.</li>
+                  error has been detected and processing is aborted.
+                  Set <a>processing mode</a>, to `json-ld-1.1`, if not already set.</li>
```

Comment:
Removing this sentence, and others like it, really would make no difference in how we actually process.